src/entity.cpp

```diff
@@ -3,6 +3,7 @@
 int currentId = 0;
```

Comment:
Probably missed this from last time but when did this get here and why isnt it a proper private variable